integration-tests/client/chainlink_models.go

```diff
@@ -904,6 +908,7 @@ p2pPeerID                              = "{{.P2PPeerID}}"
 keyBundleID                            = "{{.KeyBundleID}}"
 monitoringEndpoint                     ={{if not .MonitoringEndpoint}} "chain.link:4321" {{else}} "{{.MonitoringEndpoint}}" {{end}}
 transmitterAddress                     = "{{.TransmitterAddress}}"
+forwardingAllowed					   ={{if not .ForwardingAllowed}} false {{else}} {{.ForwardingAllowed}} {{end}}
```

Comment:
Can this be jusrt {{.ForwardingAllowed}} and make sure it defaults to false?

Reply:
thanks. updated